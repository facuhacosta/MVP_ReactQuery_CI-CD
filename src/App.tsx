import CardList from './components/CardsList/CardList'
import './App.css'

function App() {
  return (
    <div style={{display: 'flex', alignItems: 'center', flexDirection: 'column', justifyContent: 'center', width: '100%'}}>
      <h1>Personajes de Marvel</h1>
      <CardList />
    </div>
  )
}

export default App
