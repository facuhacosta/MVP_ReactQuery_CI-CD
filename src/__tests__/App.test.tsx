import App from "../App";
import { render } from "@testing-library/react"
import { expect, describe, it } from 'vitest'

describe("App test", () => {
  it("Sould match snapshot",() => {
    const { asFragment } = render(<App/>)

    expect(asFragment()).toMatchSnapshot()
  })
})