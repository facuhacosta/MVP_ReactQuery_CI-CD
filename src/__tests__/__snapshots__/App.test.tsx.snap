// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App test > Sould match snapshot 1`] = `
<DocumentFragment>
  <div
    style="display: flex; align-items: center; flex-direction: column; justify-content: center; width: 100%;"
  >
    <h1>
      Personajes de Marvel
    </h1>
    <div
      class="_container_5ea73b"
    >
      <p>
        Is Loading ...
      </p>
    </div>
  </div>
</DocumentFragment>
`;
